CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# [4.9.0](https://github.com/algolia/instantsearch.js/compare/v4.8.7...v4.9.0) (2020-12-01)
+
+
+### Bug Fixes
+
+* remove a warning about insights that is not relevant anymore ([#4593](https://github.com/algolia/instantsearch.js/issues/4593)) ([b5f6a47](https://github.com/algolia/instantsearch.js/commit/b5f6a479ff1b9b692c733f51e39eade724ff3413))
+* **breadcrumb:** add attribute to render state ([#4472](https://github.com/algolia/instantsearch.js/issues/4472)) ([ff86374](https://github.com/algolia/instantsearch.js/commit/ff8637403928a1f9c5992c05ff5ef4cc4772e448))
+* **hits:** provide `sendEvent` and `bindEvent` to `init` ([#4570](https://github.com/algolia/instantsearch.js/issues/4570)) ([44fce1c](https://github.com/algolia/instantsearch.js/commit/44fce1c00a4a6ea482899f28af7117fd9662c711))
+* **renderState:** initialize connectorState before init ([#4579](https://github.com/algolia/instantsearch.js/issues/4579)) ([91c535a](https://github.com/algolia/instantsearch.js/commit/91c535ae009f5329f4747d39b9d4d0e99d2ab07f))
+* **renderState:** initialize sendEvent before init ([#4583](https://github.com/algolia/instantsearch.js/issues/4583)) ([860998b](https://github.com/algolia/instantsearch.js/commit/860998b37ef95426707c66787dd05f16db0b1d62))
+* **types:** update WidgetFactory types ([#4567](https://github.com/algolia/instantsearch.js/issues/4567)) ([d4eb809](https://github.com/algolia/instantsearch.js/commit/d4eb80955c08df863baaced1dcb1216036dccd1c))
+* provide both `getWidgetRenderState` and `getRenderState` in connectors ([#4518](https://github.com/algolia/instantsearch.js/issues/4518)) ([d765e67](https://github.com/algolia/instantsearch.js/commit/d765e67c8362372bdfd9b78c9f394affd69de03d))
```

Comment:
I don't think we need these L7-L12 because they are the subsidiary tasks which fixed the issues from the render state PRs.

Reply:
yep, I just changed them 👍

Reply:
haha I was too slow 😂